samigo/samigo-app/src/webapp/jsf/evaluation/downloadFileSubmissions.jsp

```diff
@@ -159,7 +159,7 @@
 
   <f:verbatim><br /></f:verbatim>
   <f:verbatim><div class="tier1"></f:verbatim>
-  <h:messages infoClass="messageSamigo" warnClass="messageSamigo" errorClass="messageSamigo" fatalClass="messageSamigo"/>
+  <h:messages infoClass="sak-banner-error" warnClass="sak-banner-error" errorClass="sak-banner-error" fatalClass="sak-banner-error"/>
```

Comment:
Should the infoClass use .sak-banner-info and the warnClass use .sak-banner-warn here?

Reply:
Thanks!